File: app/src/main/java/com/zawzaw/savethelibrary/event/main/OttoBus.java
package com.zawzaw.savethelibrary.event.main;

import com.squareup.otto.Bus;
import com.squareup.otto.ThreadEnforcer;

/**
 * Created by zawzaw on 31/12/17.
 */

public class OttoBus {

    private static Bus bus;

    public static Bus getBus() {
        if (bus == null) {
            bus = new Bus(ThreadEnforcer.MAIN);
        }

        return bus;
    }

}

File: app/src/main/java/com/zawzaw/savethelibrary/event/main/OttoBus.kt
package com.zawzaw.savethelibrary.event.main

import com.squareup.otto.Bus
import com.squareup.otto.ThreadEnforcer

/**
 * Created by zawzaw on 31/12/17.
 */

object OttoBus {

    private var bus: Bus? = null

    fun getBus(): Bus {
        if (bus == null) {
            bus = Bus(ThreadEnforcer.MAIN)
        }

        return bus
    }

}
